File: programme/uppercase-to-lowercase/uppercase-to-lowercase.cpp
// Write a programme to convert uppercase string to lowercase
#include <iostream>

using namespace std;

int main()
{
    string ask;
    cout << "Input  : ";
    getline(cin, ask);
    int store;
    cout << "Output : ";
    for (auto i : ask)
    {
        store = i;
        if (store >= 65 && store <= 90)
            store += 32;
        cout << (char)store;
    }
    cout << endl;
    return 0;
}

File: programme/uppercase-to-lowercase/uppercaseToLowercase.java
// Write a programme to convert uppercase string to lowercase string.
import java.util.Scanner;

public class uppercaseToLowercase {

  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    System.out.print("Input  : ");
    String s = sc.nextLine();

    System.out.println("Output : " + s.toLowerCase());

    sc.close();
  }
}
